card/lib/generators/card/format/USAGE

```diff
@@ -0,0 +1,9 @@
+Description:
```

Comment:
I didn't think so, but maybe things got messed up from a merge or something.  It might not be a bad idea to re-create this branch fresh and see if that can be fixed.